src/org/traccar/model/Position.java

```diff
@@ -31,15 +31,16 @@
     public static final String KEY_EVENT = "event";
     public static final String KEY_ALARM = "alarm";
     public static final String KEY_STATUS = "status";
-    public static final String KEY_ODOMETER = "odometer"; // meters
-    public static final String KEY_TRIP_ODOMETER = "tripOdometer";
-    public static final String KEY_HOURS = "hours";
+    public static final String KEY_ODOMETER = "odometer";                   // kilometers
+    public static final String KEY_ODOMETER_SERVICE = "serviceOdometer";    // kilometers
+    public static final String KEY_ODOMETER_TRIP = "tripOdometer";          // kilometers
+    public static final String KEY_HOURS = "hours";                         // hours
     public static final String KEY_INPUT = "input";
     public static final String KEY_OUTPUT = "output";
-    public static final String KEY_POWER = "power";
-    public static final String KEY_BATTERY = "battery";
-    public static final String KEY_FUEL = "fuel";
-    public static final String KEY_FUEL_CONSUMPTION = "fuelConsumption";
+    public static final String KEY_POWER = "power";                         // volts
```

Comment:
I am trying to drive conformity in the values.  Without specifying a preference, work will continue with independent units of measure.  I have acknowledged the variation but propose a preference in comments in the code.